priv/templates/phx.gen.html/index.html.heex

```diff
@@ -34,6 +34,12 @@
             <%%= filter_number_input(:<%= schema.singular %>, :<%= key %>, @conn.params) %>
           </div>
         <% end %>
+        <%= for {key, type} <- schema.attrs, type in [:binary_id] do %>
+          <div class="field">
+            <label><%= Phoenix.Naming.humanize(Atom.to_string(key)) %></label>
+            <%%= filter_uuid(:<%= schema.singular %>, :<%= key %>, @conn.params) %>
```

Comment:
I believe we are missing the actual input here to compare the UUID against, no?  See #520 for reference maybe?